platforms/android/tangram/src/main/java/com/mapzen/tangram/TouchInput.java

```diff
@@ -192,10 +192,13 @@ public boolean isMultiTouch() {
     private RotateResponder rotateResponder;
     private ShoveResponder shoveResponder;
 
+    private MapController.MapChangeListener mapChangeListener;
```

Comment:
I don't think `TouchInput` should concern itself directly with map events. This class works well right now because its concerns are very cleanly separated as "identifying gestures". Actions to take in response to gestures (like signaling map events) belong in the responders. In our case, this means the logic belongs in `MapController`.

I'm guessing that incorporating this logic into `TouchInput` was done in order to capture certain events like "motion is about to start" for which we had no responder methods. I think the better solution to this is to create responder methods for these events.